### PR TITLE
Fixes #71: Added links

### LINKED ISSUE #71: Add missing profile links
###Working on it.

### PR DESCRIPTION
**Pull request by samyakbvs**
#71

### CODE REVIEW

**Review by saarthakchats — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -894,6 +904,7 @@ <h1>Featured Speakers</h1>
 											<a target="default" href="http://global.rakuten.com/corp/about/"><i class="icon fa fa-external-link"></i></a>
                                             <a target="default" href="https://github.com/alvations"><i class="icon fa fa-github"></i></a>
 											<a target="default" href="https://www.linkedin.com/in/alvations/"><i class="icon social_linkedin"></i></a>
+                                            <a target="default" href="https://twitter.com/LilingTan?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor"><i class="icon social_twitter"></i></a>
```
Comment (0x48piraj) [inline index.html]: Not a good link. Fix the `ref_src`

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Fix links as suggested

**Comment by arjunpandey2003** [comment]
Good work

**Review by arjunpandey2003 — APPROVED** [review]

**Comment by samyakbvs** [comment]
@ms10398

File: index.html [hunk 2]
```diff
@@ -466,6 +466,10 @@ <h2>Chan Cheow Hoe</h2>
 											<div class="social-links vertical-align">
 												<a target="default" href="https://cloud.google.com/"><i class="icon fa fa-external-link"></i></a>
 												<a target="default" href="https://www.linkedin.com/in/oyvindr/"><i class="icon social_linkedin"></i></a>
+                                                <a target="default" href="https://github.com/shimanokaze"><i class="icon fa fa-github"></i></a>
+                                                <a target="default" href="https://github.com/shimanokaze"><i class="icon fa fa-github"></i></a>
+
+
```
Comment (harshit98) [inline index.html]: Remove these 2 extra lines.

File: index.html [hunk 3]
```diff
@@ -562,6 +566,10 @@ <h2>Vlado Koljibabic</h2>
 										<div class="hover-state text-center preserve3d">
 											<div class="social-links vertical-align">
 												<a target="default" href="https://www.linkedin.com/in/terekhov"><i class="icon social_linkedin"></i></a>
+                                                <a target="default" href="https://github.com/andrey-terekhov"><i class="icon fa fa-github"></i></a>
+                                                <a target="default" href="https://twitter.com/ddtru?lang=en"><i class="icon social_twitter"></i></a>
+
+
```
Comment (harshit98) [inline index.html]: Remove these two extra lines from here as well and lint the code properly.

File: index.html [hunk 4]
```diff
@@ -699,7 +707,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://togaware.com/about/graham-williams/"><i class="icon fa fa-external-link"></i></a>
-
+                                            <a target="default" href="https://twitter.com/dr_grahamwill"><i class="icon social_twitter"></i></a>
```
Comment (harshit98) [inline index.html]: Code has not been linted properly.

File: index.html [hunk 5]
```diff
@@ -753,7 +761,10 @@ <h1>Featured Speakers</h1>
 											<a target="default" href="https://twitter.com/simon_riggs"><i class="icon social_twitter"></i></a>
 											<a target="default" href="https://2ndquadrant.com"><i class="icon fa fa-external-link">
 											<a target="default" href="https://uk.linkedin.com/in/simonat2ndquadrantdotcom"><i class="icon social_linkedin"></i></a>
+
+
 											<a href="https://github.com/ppaeps"><i class="icon social_github"></i></a>
+
```
Comment (harshit98) [inline index.html]: Why you're leaving space here? Remove it.

File: index.html [hunk 6]
```diff
@@ -753,7 +761,10 @@ <h1>Featured Speakers</h1>
 											<a target="default" href="https://twitter.com/simon_riggs"><i class="icon social_twitter"></i></a>
 											<a target="default" href="https://2ndquadrant.com"><i class="icon fa fa-external-link">
 											<a target="default" href="https://uk.linkedin.com/in/simonat2ndquadrantdotcom"><i class="icon social_linkedin"></i></a>
+
+
```
Comment (harshit98) [inline index.html]: Remove spaces from here as well.

File: index.html [hunk 7]
```diff
@@ -843,6 +854,7 @@ <h1>Featured Speakers</h1>
 											<a target="default" href="https://www.remlab.net/"><i class="icon fa fa-external-link"></i></a>
 
 											<a target="default" href="https://www.linkedin.com/in/remidenis/"><i class="icon social_linkedin"></i></a>
+                                            <a target="default" href="https://twitter.com/remi_denis?lang=en"><i class="icon social_twitter"></i></a>
```
Comment (harshit98) [inline index.html]: Lint the code properly.

File: index.html [hunk 8]
```diff
@@ -896,6 +908,7 @@ <h1>Featured Speakers</h1>
 											<a target="default" href="http://global.rakuten.com/corp/about/"><i class="icon fa fa-external-link"></i></a>
                                             <a target="default" href="https://github.com/alvations"><i class="icon fa fa-github"></i></a>
 											<a target="default" href="https://www.linkedin.com/in/alvations/"><i class="icon social_linkedin"></i></a>
+
```
Comment (harshit98) [inline index.html]: Remove extra space.

**Comment by harshit98** [comment]
@samyakbvs Do the requested changes. Also change your PR name to `Fixes #(issue-num): details of the fix`. Thanks!

**Review by harshit98 — APPROVED** [review]

**Pull request merged**